addon/components/listbox.js

```diff
@@ -74,6 +74,22 @@ export default class ListboxComponent extends Component {
     return true;
   }
 
+  @action
+  handleKeyDown(event) {
+    if (
+      [
+        'ArrowUp',
+        'ArrowDown',
+        'ArrowLeft',
+        'ArrowRight',
+        'Home',
+        'End',
+      ].indexOf(event.key) > -1
```

Comment:
Two tiny thoughts (neither of which are blockers here, by any means)

1. We could move this array up to the module scope to avoid creating it on every `keydown` event from the user.
2. We could make it a `Set` instead of an array, which might make the code read a little more easily

```
if (HANDLED_KEYDOWN_EVENTS.has(event.key)) {
  ...
}
```

Reply:
👍 Agreed. Much cleaner and more performant. I've fixed that here: https://github.com/GavinJoyce/ember-headlessui/pull/81/commits/6f34df60eb080085546832ac1fbb291e867a28fa